scripts/globals/magic_utils/spell_damage.lua

```diff
@@ -206,7 +206,10 @@ end
 -- Careful description required to prevent further problems where people misunderstand what is happening.
 -- Multiple things can affect dmg. in our core SDT is a % reduction in its own step.
 xi.magic_utils.spell_damage.calculateSDT = function(caster, target, spell, spellElement)
-    local SDT = target:getMod(xi.magic.specificDmgTakenMod[spellElement]) -- The variable we want to calculate
+    local SDT    = 1 -- The variable we want to calculate
+    local SDTMod = target:getMod(xi.magic.specificDmgTakenMod[spellElement]) -- Elemental SDT mods are Base 100.
+
+    SDT = (SDTMod / -100) + 1 -- Convert Modifier value.
```

Comment:
like I was saying on discord, the sql has to do that to create modifiers. you do not want to repeat it and they are -not- base 100 they are base 1000 to support half percent

From modifier.h and status.lua:
```
Example: 1000 = 100%, 875= 87.5%
```
this confusion is precisely why I left that comment there